File: Chapter 03/Java/src/main/java/com/grokkingsoftwarearchitecture/chapter03/section_3_4_order_processor_refactor/after/Order.java
package com.grokkingsoftwarearchitecture.chapter03.section_3_4_order_processor_refactor.after;

import java.util.List;

/**
 * A Simple Data Transfer Object (DTO).
 * * ARCHITECTURE NOTE: Just like in the "Before" state, this remains a pure 
 * data container. It holds the state that our new, highly-cohesive services 
 * will operate on.
 */
public class Order {
    public List<String> items;
    public double total;
    public String customerEmail;

    public Order(List<String> items, double total, String customerEmail) {
        this.items = items;
        this.total = total;
        this.customerEmail = customerEmail;
    }
}

File: Chapter 03/C#/Section_3_4_OrderProcessor_Refactor/After/Order.cs
using System.Collections.Generic;

namespace Chapter03.OrderProcessorRefactor.After;

/// <summary>
/// A Simple Data Transfer Object (DTO).
/// 
/// ARCHITECTURE NOTE: Just like in the "Before" state, this remains a pure 
/// data container. It holds the state that our new, highly-cohesive services 
/// will operate on.
/// </summary>
public class Order
{
    public required List<string> Items { get; set; }
    public decimal Total { get; set; }
    public required string CustomerEmail { get; set; }
}